src/dataEntryApp/views/subjectDashBoard/components/subjectDashboardGeneralTab.js

```diff
@@ -0,0 +1,179 @@
+import React, { Fragment } from "react";
+import { Paper } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Button from "@material-ui/core/Button";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import Typography from "@material-ui/core/Typography";
+import Fab from "@material-ui/core/Fab";
+import moment from "moment/moment";
+
+const useStyles = makeStyles(theme => ({
+  label: {
+    marginLeft: theme.spacing(2),
+    marginBottom: theme.spacing(2)
+  },
+  enrollButtonStyle: {
+    marginBottom: theme.spacing(2)
+  },
+  root: {
+    flexGrow: 1,
+    padding: theme.spacing(2)
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "left"
+  },
+  programStatusStyle: {
+    color: "red",
+    backgroundColor: "#FFB6C1",
+    borderRadius: "5px"
+  },
+  expansionHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    flexBasis: "33.33%",
+    flexShrink: 0
+  },
+  listItem: {
+    paddingBottom: "0px",
+    paddingTop: "0px"
+  }
+}));
+
+const date1 = new Date();
```

Comment:
date1 does not convey what this variable does. Can you please give it a meaningful name.